File: unittest/scripts/devapi/scripts/document_identity_2_3.js
// Assumptions: test schema assigned to db, my_collection collection exists

// using a books unique ISBN as the object ID
var myColl = db.getCollection('my_collection');

myColl.add( {
        _id: "978-1449374020",
        title: "MySQL Cookbook: Solutions for Database Developers and Administrators"
}).execute();

var book = myColl.find('_id = "978-1449374020"').execute();

File: unittest/scripts/py_dev_api_examples/scripts/document_identity_2_3.py
# Assumptions: test schema assigned to db, my_collection collection exists

# using a books unique ISBN as the object ID
myColl = db.getCollection('my_collection')

myColl.add( {
        '_id': "978-1449374020",
        'title': "MySQL Cookbook: Solutions for Database Developers and Administrators"
}).execute()

book = myColl.find('_id = "978-1449374020"').execute()